src/sync-accumulator.ts

```diff
@@ -134,12 +135,36 @@ interface IAccountData {
     events: IMinimalEvent[];
 }
 
+/** A to-devive message as received from the sync. */
 export interface IToDeviceEvent {
     content: IContent;
     sender: string;
     type: string;
 }
 
+/**
+ * Successfully processed to-device message.
+ */
+export type IToDeviceMessage = IToDeviceEvent;
```

Comment:
What is the purpose of this alias? The compiler won't complain if you use the wrong one, right?

Also, what is the difference in meaning between `IToDeviceMessage` and `IToDeviceEvent`.

(Also, do we still use `I` prefix for interfaces?)

Reply:
I am trying to match a bit more the rust sdk terminlogy that uses `message` and not `event` for to-device (they don't have event_ids, and other properties compared to events).
Unfortunatly the spec uses a mix of message and event for to-device. 
So I kept `IToDeviceEvent` for the one coming down the sync, and `IToDeviceMessage` for the one succesfully processed (and decrypted if applicable)